static/app/components/onboardingWizard/newSidebar.tsx

```diff
@@ -0,0 +1,270 @@
+import {Fragment, useState} from 'react';
+import {css} from '@emotion/react';
+import styled from '@emotion/styled';
+
+import {Chevron} from 'sentry/components/chevron';
+import InteractionStateLayer from 'sentry/components/interactionStateLayer';
+import ProgressRing from 'sentry/components/progressRing';
+import SidebarPanel from 'sentry/components/sidebar/sidebarPanel';
+import type {CommonSidebarProps} from 'sentry/components/sidebar/types';
+import {Tooltip} from 'sentry/components/tooltip';
+import {IconCheckmark} from 'sentry/icons';
+import {t, tct} from 'sentry/locale';
+import pulsingIndicatorStyles from 'sentry/styles/pulsingIndicator';
+import {space} from 'sentry/styles/space';
+import {isDemoWalkthrough} from 'sentry/utils/demoMode';
+
+function getPanelDescription(walkthrough: boolean) {
+  if (walkthrough) {
+    return {
+      title: t('Guided Tours'),
+      description: t('Take a guided tour to see what Sentry can do for you'),
+    };
+  }
+  return {
+    title: t('Quick Start'),
+    description: t('Walk through this guide to get the most out of Sentry right away.'),
+  };
+}
+
+interface TaskProps {
+  description: string;
+  title: string;
+  status?: 'waiting' | 'completed';
+}
+
+function Task({title, description, status}: TaskProps) {
+  if (status === 'completed') {
+    return (
+      <TaskWrapper completed>
+        <strong>{title}</strong>
+        <IconCheckmark color="green300" isCircled />
+      </TaskWrapper>
+    );
+  }
+
+  if (status === 'waiting') {
+    return (
+      <TaskWrapper>
+        <InteractionStateLayer />
+        <div>
+          <strong>{title}</strong>
+          <p>{description}</p>
+        </div>
+        <Tooltip title={t('Waiting for event')}>
+          <PulsingIndicator />
+        </Tooltip>
+      </TaskWrapper>
+    );
+  }
+
+  return (
+    <TaskWrapper>
+      <InteractionStateLayer />
+      <div>
+        <strong>{title}</strong>
+        <p>{description}</p>
+      </div>
+    </TaskWrapper>
+  );
+}
+
+interface TaskGroupProps {
+  description: string;
+  title: string;
+  totalCompletedTasks: number;
+  totalTasks: number;
+  expanded?: boolean;
+}
+
+function TaskGroup({
+  title,
+  description,
+  totalCompletedTasks,
+  totalTasks,
+  expanded,
+}: TaskGroupProps) {
+  const [isExpanded, setIsExpanded] = useState(expanded);
+  return (
+    <TaskGroupWrapper>
+      <TaskGroupHeader onClick={() => setIsExpanded(!isExpanded)}>
+        <InteractionStateLayer />
+        <span>
+          <strong>{title}</strong>
+          <p>{description}</p>
+        </span>
```

Comment:
nesting a block element (`p`) inside an inline element (`span`) is not valid HTML.
I would suggest to use a `div` as the outer element or change the `p` to also be a `span`.

Reply:
oh yes... I probably did not see this error in the console... will update it in a follow-up